recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - make
     - pkg-config
   host:
-    - boost
+    - libboost-python-devel
```

Comment:
It seems we could actually use just `libboost-headers` here, as none of the libraries are necessary at runtime:
```
WARNING (yosys): dso library package conda-forge::libboost-1.82.0-h6fcfa73_5 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (yosys): dso library package conda-forge::libboost-python-1.82.0-py311h92ebd52_5 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```